.github/ISSUE_TEMPLATE/new-model-addition.yml

```diff
@@ -0,0 +1,31 @@
+name: "\U0001F31F New model addition"
+description: Submit a proposal/request to implement a new model
+labels: [ "New model" ]
```

Comment:
Same as above regarding the label 'new model' appearing under 'new model addition' like 'bug' in 'bug report'.